tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/IoDataGenerationTest.java

```diff
@@ -275,4 +290,78 @@ else if (e.label().equals("writtenBy"))
         GraphSONWriter.build().mapper(GraphSONMapper.build().embedTypes(true).create()).create().writeGraph(os4, g);
         os4.close();
     }
+
+    @Test
+    public void shouldWriteGratefulDeadGraphSONV2d0() throws IOException {
+        final TinkerGraph g = TinkerGraph.open();
+        final TinkerGraph readG = TinkerGraph.open();
+
+        final GraphReader reader = GryoReader.build().create();
+        try (final InputStream stream = AbstractGremlinTest.class.getResourceAsStream("/org/apache/tinkerpop/gremlin/structure/io/gryo/grateful-dead.kryo")) {
+            reader.readGraph(stream, g);
+        }
+        final OutputStream os2 = new FileOutputStream(tempPath + "grateful-dead-V2d0-typed.json");
+        GraphSONWriter.build().mapper(GraphSONMapper.build().version(GraphSONVersion.V2_0).typeInfo(GraphSONMapper.TypeInfo.PARTIAL_TYPES).create()).create().writeGraph(os2, g);
+        os2.close();
+
+        final InputStream is = new FileInputStream(tempPath + "grateful-dead-V2d0-typed.json");
+        GraphSONReader.build().mapper(GraphSONMapper.build().version(GraphSONVersion.V2_0).typeInfo(GraphSONMapper.TypeInfo.PARTIAL_TYPES).create()).create().readGraph(is, readG);
+        is.close();
+
+        assertEquals(approximateGraphsCheck(g, readG), true);
+    }
+
+    /**
+     * Checks sequentially vertices and egdes of both graphs. Will check sequentially Vertex IDs, Vertex Properties IDs
+     * and values and classes. Then same for edges. To use when serializing a Graph and deserializing the supposedly
+     * same Graph.
+     */
+    private boolean approximateGraphsCheck(Graph g1, Graph g2) {
```

Comment:
Ah, looking at the pom, I'm getting this warning during the build : 

```
[INFO] --- maven-resources-plugin:2.6:copy-resources (copy-graphson-from-tmp-to-resources) @ tinkergraph-gremlin ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/kevingallardo/Documents/workspace/newkek-incubator-tinkerpop/tinkergraph-gremlin/target/tinkerpop-io
```
Seems like the pom is going to search for the ressources in `<io.tmp.dir>${project.build.directory}/tinkerpop-io</io.tmp.dir>` and `${project.build.directory}` is defined as `<directory>${basedir}/target</directory>`.
But the `tempPath` in IoDataGenerationTest` is `tempPath = TestHelper.makeTestDataPath(TinkerGraphTest.class, "tinkerpop-io").getPath() + File.separator;`

If I change the pom to search in the right directory : `<io.tmp.dir>${project.build.directory}/test-case-data/TinkerGraphTest/tinkerpop-io</io.tmp.dir>`, it works.